src/app/view/components/_common/dot-push-publish-dialog/dot-push-publish-dialog.component.ts

```diff
@@ -116,6 +127,37 @@ export class DotPushPublishDialogComponent implements OnInit, OnDestroy {
         this.formEl.ngSubmit.emit();
     }
 
+    private loadCustomCode(): void {
```

Comment:
Isn't this a exact copy of the code from the link I gave you? can we extract this in some util file?